client/main/variableeditor.js

```diff
@@ -221,15 +221,16 @@ const EditorWidget = Backbone.View.extend({
         this.$el.empty();
         this.$el.addClass('silky-variable-editor-widget');
 
-        this.$title = $('<input class="silky-variable-editor-widget-title" type="text">').appendTo(this.$el);
+        this.$title = $('<input class="silky-variable-editor-widget-title" type="text" maxlength="50">').appendTo(this.$el);
         this._currentKeyboardContext = '';
         this.$title.focus(() => {
             this._currentKeyboardContext = keyboardJS.getContext();
             keyboardJS.setContext('');
             this.$title.select();
         } );
         this.$title.on("change keyup paste", () => {
-            this.model.set({ name: this.$title.val() });
+            let newName = this.$title.val().trim();
+            this.model.set({ name: newName });
         } );
```

Comment:
The enter clicking happens in the widget but the clicking the tick happens in the parent. I can pass the parent down to the widget so it can click the tick but it feels a bit dirty. Thoughts?